crates/rv/src/commands/shell/init.rs

```diff
@@ -25,5 +25,25 @@ pub fn init(config: &Config, shell: Shell) -> Result<()> {
             );
             Ok(())
         }
+        Shell::Bash => {
+            print!(
+                concat!(
+                    "_rv_autoload_hook() {{\n",
+                    "    eval \"$({} shell env bash)\"\n",
+                    "}}\n",
+                    "_rv_autoload_hook\n",
+                    "_chpwd_hook() {{\n",
```

Comment:
Should this be prefixed with `rv` so the function doesn't clash with other hooks the user might have defined?

Reply:
Sounds good to me.